include/mbedtls/config.h

```diff
@@ -484,6 +484,11 @@
  * is still present and it is used for group structures not supported by the
  * alternative.
  *
+ * The original implementation can in addition be removed by setting the
+ * MBEDTLS_ECP_NO_FALLBACK, in which case any function for which the
+ * corresponding MBEDTLS_ECP__FUNCTION_NAME__ALT macro is defined will not be
```

Comment:
Minor: `MBEDTLS_ECP__FUNCTION_NAME__ALT` is typeset literally with the double underscores. This is preexisting so I'm not going to request that you find the correct Doxygen syntax to put the `FUNCTION_NAME` part in a different font.

Reply:
Pre-existing in other places in config.h too, so leaving this for a batch cleanup operation for another time.